openedx/core/djangoapps/credit/api/eligibility.py

```diff
@@ -284,6 +284,65 @@ def set_credit_requirement_status(username, course_key, req_namespace, req_name,
                 log.error("Error sending email")
 
 
+# pylint: disable=invalid-name
+def remove_credit_requirement_status(username, course_key, req_namespace, req_name):
+    """
+    Remove the user's requirement status.
+
+    This will remove the record from the credit requirement status table.
+    The user will still be eligible for the credit in a course.
+
+    Args:
+        username (str): Username of the user
+        course_key (CourseKey): Identifier for the course associated with the requirement.
+        req_namespace (str): Namespace of the requirement (e.g. "grade" or "reverification")
+        req_name (str): Name of the requirement (e.g. "grade" or the location of the ICRV XBlock)
```

Comment:
Doc string line length should not be more than 80 characters as per https://github.com/edx/edx-platform/wiki/Python-Guidelines#follow-pep-8